### PR TITLE
Add store status response to statusServer.

### PR DESCRIPTION
**Pull request by BramGruneir**
This is very similar to how node statuses are handled.
Next up is dealing with requesting a single node/store status.

### CODE REVIEW

File: server/status.go [hunk 1]
```diff
@@ -86,8 +90,8 @@ func (s *statusServer) registerHandlers(mux *http.ServeMux) {
 	mux.HandleFunc(statusGossipKeyPrefix, s.handleGossipStatus)
 	mux.HandleFunc(statusLocalKeyPrefix, s.handleLocalStatus)
 	mux.HandleFunc(statusLocalStacksKey, s.handleLocalStacks)
-	mux.HandleFunc(statusNodesKeyPrefix, s.handleNodesStatus)
-	mux.HandleFunc(statusStoresKeyPrefix, s.handleStoresStatus)
+	mux.HandleFunc(statusNodesKeyPrefix, s.handleNodeStatus)
```
Comment (spencerkimball) [inline server/status.go]: Want to change the key prefixes to be singular as well for consistency?

File: server/status.go [hunk 2]
```diff
@@ -150,7 +154,7 @@ func (s *statusServer) handleLocalStacks(w http.ResponseWriter, r *http.Request)
 	}
 }
 
-// handleNodesStatus handles GET requests for all node status.
+// handleNodesStatus handles GET requests for all node's statuses.
```
Comment (spencerkimball) [inline server/status.go]: s/node's/node/

File: server/status.go [hunk 3]
```diff
@@ -188,16 +192,58 @@ func (s *statusServer) handleNodesStatus(w http.ResponseWriter, r *http.Request)
...
 
-// handleStoresStatus handles GET requests for store status.
+// handleStoresStatus handles GET requests for all store's statuses.
 func (s *statusServer) handleStoresStatus(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-	w.Write([]byte(`{"stores": []}`))
+	startKey := engine.KeyStatusStorePrefix
+	endKey := startKey.PrefixEnd()
+
+	call := client.Scan(startKey, endKey, 0)
```
Comment (spencerkimball) [inline server/status.go]: Do you think it's worthwhile to have a single handler helper function which handles nodes and stores but accepts the applicable key prefix? These two sorts of status are probably going to remain quite equivalent.

File: server/status_test.go [hunk 4]
```diff
@@ -277,3 +277,74 @@ func TestNodeStatusResponse(t *testing.T) {
 		t.Errorf("node status descriptors are not equal\nexpected:%+v\nactual:%+v\n", ts.node.Descriptor, nodeStatuses[0].Desc)
 	}
 }
+
+// TestStoreStatusResponse verifies that node status returns the expected
+// results.
+// TODO(Bram): Add more nodes.
+func TestStoreStatusResponse(t *testing.T) {
```
Comment (spencerkimball) [inline server/status_test.go]: Might be worth sharing some of the test infrastructure as well.

**Comment by spencerkimball** [comment]
LGTM

File: server/status.go (hunk 1 above)
Comment (BramGruneir) [inline server/status.go]: Done.

File: server/status.go (hunk 2 above)
Comment (BramGruneir) [inline server/status.go]: fixed

File: server/status.go (hunk 3 above)
Comment (BramGruneir) [inline server/status.go]: Good idea.  I'll address this in another PR.

File: server/status_test.go (hunk 4 above)
Comment (BramGruneir) [inline server/status_test.go]: done

**Pull request merged**